commitizen/commands/changelog.py

```diff
@@ -114,28 +114,28 @@ def _find_incremental_rev(self, latest_version: str, tags: list[GitTag]) -> str:
         on our experience.
         """
         SIMILARITY_THRESHOLD = 0.89
-        tag_ratio = map(
-            lambda tag: (
-                SequenceMatcher(
+        items = (
```

Comment:
```suggestion
        scores_and_tag_names: Generator[tuple[int, str]] = (
```

might be a better name. `items` is too broad